ckanext/scheming/templates/scheming/display_snippets/multiple_choice.html

```diff
@@ -10,9 +10,12 @@
 {%- if labels|length == 1 -%}
   {{ labels[0] }}
 {%- else -%}
-  <ul>
+    {%- if 'sorted_choices' in field -%}
```

Comment:
this should be `if field.get('sorted_choices')` in otherwise the choices will be sorted when a user enters `sorted_choices: false`